packages/web/src/graphql/withCell.tsx

```diff
@@ -43,38 +62,47 @@ export const withCell = ({
   }),
   QUERY,
   afterQuery = (data) => ({ ...data }),
-  Loading = () => 'Loading...',
+  Loading = () => <div>Loading...</div>,
```

Comment:
I had to re-add the `div` element otherwise it would keep saying that the returned string isn't a valid React component :| Is there a type that would be more appropriate than `React.FC`?

Reply:
Would a fragment work here `<>Loading...</>`?

Reply:
Seems like it works 👍